radixdlt/src/main/java/com/radixdlt/network/transport/udp/NatHandlerRemoteImpl.java

```diff
@@ -187,6 +150,31 @@ boolean endValidation(ByteBuf buf) {
 		return true;
 	}
 
+	@Override
+	public int computeSize(InetAddress destAddress) {
+		byte[] rawSourceAddress = getAddress().getAddress();
+		byte[] rawDestAddress = destAddress.getAddress();
+
+		assert rawSourceAddress.length == 4 || rawSourceAddress.length == 16;
```

Comment:
Is this to check that the addresses are IPv4/IPv6?

Reply:
One or the other, yes.  In case there is some new scheme introduced that we are not ready for.